Update condition types for Function CR (#8853)
kyma-project/kyma: docs/serverless/06-01-function-cr.md

| -------------------------------- | -------------------- | ------------------------------------------------------------------------------------------------------------------ |
| `ConfigMapCreated`               | `ConfigurationReady` | A new ConfigMap was created based on the Function CR definition.                                                   |
| `ConfigMapUpdated`               | `ConfigurationReady` | The existing ConfigMap was updated after changes in the Function CR name, its source code or dependencies.         |
| `JobFailed`                      | `BuildReady`         | The image with the Function's configuration could not be created due to an error.                                  |
| `JobCreated`                     | `BuildReady`         | The Kubernetes Job resource that builds the Function image was created.                                            |

> | `JobUpdated`                     | `BuildReady`         | The existing Job was updated after changing the Function's metadata and spec fields, which are not affecting the way of building the Function image, like labels. |

Reviewer: ```suggestion
| `JobUpdated`                     | `BuildReady`         | The existing Job was updated after changing the Function's metadata or spec fields which do not affect the way of building the Function image, such as labels.|
```
